src/dvclive/live.py

```diff
@@ -257,3 +259,9 @@ def read_step(self):
     def read_latest(self):
         with open(self.metrics_file, encoding="utf-8") as fobj:
             return json.load(fobj)
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.end()
```

Comment:
Minor follow-up: let's either make `end()` public or get rid of it and move the logic here. What do you think @daavoo?

Reply:
I slightly prefer getting rid of `end()` since I'm not sure it's worth adding to docs, but no strong preference